analyses/snv-callers/vaf_cutoff_experiment.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "VAF Cutoffs and TMB Calculation"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: C. Savonen for ALSF CCDL
+date: 2019
+---
+
+Purpose: How do the TMB calculations change with adjustments to the VAF cutoff
+filter? 
+
+### Summary of Findings:
+
```

Comment:
Your preliminary conclusion seems like it would fit well here. Unless you were waiting for more conclusive results before updating this section.

Reply:
I'll add some more explanation of the findings here.